internal/providers/terraform/azure/testdata/iothub_test/iothub_test.tf

```diff
@@ -0,0 +1,41 @@
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+# Add example resources for Iothub below
+resource "azurerm_iothub" "example" {
+  name                = "Example-IoTHub"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  sku {
+    name     = "S1"
+    capacity = 2
+  }
+
+  tags = {
+    purpose = "testing"
+  }
+}
+
+resource "azurerm_iothub_dps" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  allocation_policy   = "Hashed"
+
+  sku {
+    name     = "S1"
+    capacity = 1
```

Comment:
I saw your question about DPS' capacity. I couldn't find anything explicit about this too, but we probably still should multiply the number of operations by this capacity. My guess would be that capacity is related to IoT Hub units that are created along with this resource, because `linked_hubs` block is optional.

Reply:
https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service?WT.mc_id=Portal-Microsoft_Azure_IotHub#linked-iot-hubs

I don't think capacity is tied to the linked_hubs in any way. They would be created as separate resources and then linked in.

Reply:
Sounds good. I guess we could mention in usage param comment that users should account the capacity in the `monthly_operations` value. At least they will be aware.